code/modules/clothing/spacesuits/rig/rig_pieces_vr.dm

```diff
@@ -43,3 +43,13 @@
 
 /obj/item/clothing/suit/space/rig
 	species_restricted = list(SPECIES_HUMAN, SPECIES_SKRELL, SPECIES_TAJ, SPECIES_UNATHI, SPECIES_NEVREAN, SPECIES_AKULA, SPECIES_SERGAL, SPECIES_ZORREN_HIGH, SPECIES_VULPKANIN, SPECIES_PROMETHEAN, SPECIES_VOX, SPECIES_TESHARI, SPECIES_VASILISSAN, SPECIES_RAPALA, SPECIES_ALRAUNE, SPECIES_FENNEC, SPECIES_XENOHYBRID)
+
+/obj/item/clothing/shoes/magboots/rig/ce
+	name = "advanced boots"
```

Comment:
probably should still be named "advanced magboots" here tbh

Reply:
I think they get renamed regardless by the suit